app/controllers/form_controller.rb

```diff
@@ -146,6 +146,17 @@ def referrer_from_query
     parsed_params["referrer"][0]
   end
 
+  def original_duplicate_log_id_from_query
+    query_params = URI.parse(request.url).query
+
+    return unless query_params
+
+    parsed_params = CGI.parse(query_params)
+    return unless parsed_params["original_log_id"]
+
+    parsed_params["original_log_id"][0]
```

Comment:
if you want to play code golf
```suggestion
    parsed_params["original_log_id"]&.first
```